contracts/src/main/kotlin/com/r3/corda/lib/tokens/contracts/utilities/TransactionUtilities.kt

```diff
@@ -84,31 +84,28 @@ internal val NULL_SECURE_HASH = SecureHash.zeroHash
  * If the [TokenType] is not a [TokenPointer] this function discovers the JAR which implements the receiving [TokenType].
  */
 fun TokenType.getAttachmentIdForGenericParam(): SecureHash? {
-    val computedValue = synchronized(attachmentCache) {
-        val startingPoint = if (this is IssuedTokenType) {
-            this.tokenType.javaClass
-        } else {
-            this.javaClass
-        }
-        attachmentCache.computeIfAbsent(startingPoint) { clazz ->
-            var classToSearch: Class<*> = clazz
-            while (classToSearch != this.tokenClass && classToSearch != TokenPointer::class.java) {
-                classToSearch = this.tokenClass
-            }
-            if (classToSearch.location == TokenType::class.java.location) {
-                TokenUtilities.logger.debug("${this.javaClass} is provided by tokens-sdk")
-                NULL_SECURE_HASH
-            } else {
-                val hash = classToSearch.location.readBytes().sha256()
-                TokenUtilities.logger.debug("looking for jar which provides: $classToSearch FOUND AT: " +
-                        "${classToSearch.location.path} with hash $hash")
-                hash
-            }
-        }
-    }
-    return if (computedValue == NULL_SECURE_HASH) {
-        null
-    } else {
-        computedValue
-    }
+	val computedValue = synchronized(attachmentCache) {
```

Comment:
same here - was this intentional indenting?

Reply:
yarp! There is a code style committed into the repo - I was actually considering reformatting everything to make it compliant with the style in one big bang - but it was over 200 files. 

anyone working on tokens within intellij _should_ automatically pick up the style.